firecrawl_handler.py

```
@@ -0,0 +1,78 @@
+        # Provide more detailed error information
+        error_message = str(e)
+        if hasattr(e, 'response') and e.response:
+            status_code = getattr(e.response, 'status_code', 'unknown')
+            error_message = f"HTTP Error {status_code}: {error_message}"
+            
+            # Try to extract more details from the response if available
+            try:
+                response_text = e.response.text
+                if response_text:
+                    error_message += f" - Response: {response_text[:200]}"
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)